common/archiver/interface.go

```diff
@@ -81,29 +73,18 @@ type (
 
 	// HistoryArchiver is used to archive history and read archived history
 	HistoryArchiver interface {
-		Archive(ctx context.Context, URI string, request *ArchiveHistoryRequest, opts ...ArchiveOption) error
-		Get(ctx context.Context, URI string, request *GetHistoryRequest) (*GetHistoryResponse, error)
-		ValidateURI(URI string) error
+		Archive(ctx context.Context, URI URI, request *ArchiveHistoryRequest, opts ...ArchiveOption) error
```

Comment:
Lets make these all unnamed variables, naming is useful when its not clear what the interface does without names but here everything is clear without names.